geoconvert/convert.py

```diff
@@ -132,7 +134,7 @@ def ca_province_name_to_province_code(text):
 # GERMANY
 
 
-def de_address_to_land_code(text):
+def de_safe_address_to_land_code(text):
```

Comment:
I thought about it, and the search for the land code in plain text should not be considered as safe. So, instead, of defining a `de_safe_address_to_land_code` function, you should just add the postcode search to `de_address_to_land_code` (after the land haupstatdt in plain text, I would say).